### DISCUSSION TITLE
Why not a web address that resolves to a valid Cloudflare IP address instead of entering IP address itself?

### DISCUSSION THREAD

**Discussion by armanshahrisa** [GFW-knocker/gfw_resist_tls_proxy#96] [Ideas]
I have noticed that it is mandatory to provide an IP address that belongs to Cloudflare in order to use the script. However, using a host address that could be resolved to a valid Cloudflare IP address instead, could be a good idea. For example, dl.test.com could point to 45.131.4.112. This allows the user to change the corresponding IP address in a DNS record and the IP address that gfw_resist_tls_proxy uses can be changed in one move across multiple devices. Using a web address that resolves to a valid IP address is compatible with native v2rayN and v2rayNG. However, it doesn't work with gfw_resist_tls_proxy or v2rayF.

**Comment by GFW-knocker** [comment]
of course you can use a domain instead of ip
& it works in pyprox as well.
just replace this line with any domain that resolve to cloudflare ip:
Cloudflare_IP = "medium.com" #"discord.com" #"plos.org" #'162.159.135.42'   # plos.org (can be any dirty cloudflare ip or website)

**Comment by armanshahrisa** [comment]
Yes, it works for gfw_resist_tls_proxy, but not for v2rayF. I think it didn't work in the previous gfw_resist_tls_proxy release. That's why I suggested it without checking it again with the newer release.
